crates/volta-core/src/project/mod.rs

```diff
@@ -211,6 +212,22 @@ impl Project {
         }
     }
 
+    /// Pins the Pnpm version in this project's manifest file
+    pub fn pin_pnpm(&mut self, version: Option<Version>) -> Fallible<()> {
+        if let Some(platform) = self.platform.as_mut() {
+            update_manifest(&self.manifest_file, ManifestKey::Pnpm, version.as_ref())?;
+
+            platform.pnpm = version;
+
+            Ok(())
+        } else {
+            Err(ErrorKind::NoPinnedNodeVersion {
+                tool: "Pnpm".into(),
```

Comment:
minor: We should make sure to be consistent (both internally and with `pnpm` branding) in our capitalization. I believe there are some cases that use `Pnpm` while others use `pnpm`. From their website, it looks like `pnpm` uses the same all-lowercase branding that `npm` does, so we should probably match that everywhere.